core/sawtooth_poet/poet_consensus/poet_fork_resolver.py

```diff
@@ -243,6 +244,15 @@ def compare_forks(self, cur_fork_head, new_fork_head):
                     new_fork_head.header_signature[:8],
                     cur_fork_head.header_signature[:8])
                 chosen_fork_head = new_fork_head
+            else:
+                LOGGER.info(
+                    'Choose current fork %s over new fork %s: '
+                    'Current fork header signature (%s) is same as new fork '
+                    'header signature (%s)',
+                    cur_fork_head.header_signature[:8],
+                    new_fork_head.header_signature[:8],
+                    cur_fork_head.header_signature[:8],
+                    new_fork_head.header_signature[:8])
```

Comment:
you might drop the redundant second pair of signature outputs to make the log message more compact.

Reply:
Done